files/smail.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
+
+# Copyright 2015 Princeton University Research Computing
```

Comment:
license?

Reply:
It's in the Slurm repo, under contrib. So not sure what the exact license is here. But it's certainly free to use, otherwise it would not have been there, I assume.

Reply:
can you add that here in a comment? incl the commit you used